File: src/main/java/com/google/android/finsky/api/DfeServerError.java
package com.google.android.finsky.api;

import com.android.volley.VolleyError;

class DfeServerError extends VolleyError {

    String mDisplayErrorHtml;

    DfeServerError(String displayErrorHtml) {
        super();

        mDisplayErrorHtml = displayErrorHtml;
    }

    public String getDisplayErrorHtml() {
        return this.mDisplayErrorHtml;
    }

    @Override
    public String toString() {
        return "DisplayErrorMessage[" + this.mDisplayErrorHtml + "]";
    }
}

File: playstore/src/main/java/finsky/api/DfeServerError.kt
package finsky.api


open class DfeError(message: String, cause: Throwable? = null): Exception(message, cause) {

}

class DfeParseError(message: String, cause: Throwable? = null) : DfeError(message, cause) {

}

class DfeServerError(message: String) : DfeError(message) {
    override fun toString(): String {
        return "DisplayErrorMessage[$message]"
    }
}